restapi/src/main/java/io/stargate/web/docsapi/dao/DocumentDB.java

```diff
@@ -64,26 +69,33 @@
   private final AuthenticationSubject authenticationSubject;
 
   static {
+    allColumns = new ArrayList<>();
     allColumnNames = new ArrayList<>();
     allColumnTypes = new ArrayList<>();
     allPathColumnNames = new ArrayList<>();
     allPathColumnTypes = new ArrayList<>();
     allColumnNames.add("key");
     allColumnTypes.add(Type.Text);
+    allColumns.add(Column.create("key", Type.Text));
     for (int i = 0; i < MAX_DEPTH; i++) {
       allPathColumnNames.add("p" + i);
       allPathColumnTypes.add(Type.Text);
+      allColumns.add(Column.create("p" + i, Type.Text));
     }
     allColumnNames.addAll(allPathColumnNames);
     allColumnTypes.addAll(allPathColumnTypes);
     allColumnNames.add("leaf");
     allColumnTypes.add(Type.Text);
+    allColumns.add(Column.create("leaf", Type.Text));
```

Comment:
you may change the ordering of opeartions to:
```
Column leafColumn = Column.create("leaf", Type.Text);
allColumns.add(leafColumn);
allColumnNames.add(leafColumn.name());
allColumnTypes.add(leafColumn.type());
```
to reduce the duplication a bit.